### DISCUSSION TITLE
Welcome to CPI-Helper-Chrome-Extension Discussions!

### DISCUSSION THREAD

**Discussion by dbeck121** [dbeck121/CPI-Helper-Chrome-Extension#211] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sn6789** [comment]
We are using NS2 cloud version of BTP Integration Suite.  How do I enable CPI Helper for NS2 IS subaccounts?

**Comment by dbeck121** [comment]
Hi,

ns2 cloud has a very special url and the addon is not compatible with it. If you want to enable it, you must take the ressources and adjust the urls in manifest file. Then you need to load it in chrome as unzipped extension (for developers). Make sure your organisation allows the use of CPI-Helper first. You also won't get any updates.

**Comment by sn6789** [comment]
Thank you for your prompt response.  Will try the solution.

**Comment by horstdfrank-sys** [comment]
Hey Guys,
your Ext ist really good. But in some Env. I get issues nowadays.

CPI Helper fails for some users because calls to /api/v1/IntegrationRuntimeArtifacts return 401 Unauthorized with WWW-Authenticate: Basic. The CPI UI works (SSO session active), but the Runtime API does not accept the session and requires Basic Auth. This results in failing requests and broken functionality in Monitoring/Security Material views. Other users with similar roles do not have this issue, so it seems to be an authentication inconsistency rather than a missing role or extension problem.

Please help me if you perhabs know where the issue is. 

BR
Franky

**Reply by dbeck121** [reply]
Hey!

Please Check https://github.com/dbeck121/CPI-Helper-Chrome-Extension/issues/320#issuecomment-4509005912

Does that help?

This is because of old url structures. To avoid this in the future, please think about voting for https://influence.sap.com/sap/ino/#/idea/336795/?section=sectionDetails

**Reply by horstdfrank-sys** [reply]
Hi @dbeck121,
thx for response.
I try it and come to a overview Page and click on this url but its the same issue like before 
<img width="742" height="289" alt="Screenshot 2026-06-11 111401" src="https://github.com/user-attachments/assets/a517f0fd-d038-44a4-93be-00b6751e8335" />
<img width="1108" height="859" alt="Screenshot 2026-06-11 111554" src="https://github.com/user-attachments/assets/1e0563ae-ccc4-458c-8366-f09358755674" />
BR
Franky

**Reply by dbeck121** [reply]
@horstdfrank-sys does the url have integrationsuite in its host?

**Reply by horstdfrank-sys** [reply]
Hi @dbeck121,
The API Call that your extension is try to call is this
https://....... ...cfapps.eu10.hana.ondemand.com/api/v1/IntegrationRuntimeArtifacts(%27DEV_HUB_D365_S4%27)?$format=json
then it comes to the basci Auth.

Sadly no.

<img width="1241" height="213" alt="image" src="https://github.com/user-attachments/assets/ed35e195-3fa8-416f-bbf6-3ddaa454c2c7" />

BR
Franky

**Reply by dbeck121** [reply]
Use this comment to create url yourself https://github.com/dbeck121/CPI-Helper-Chrome-Extension/issues/320#issuecomment-4386963676

This button should normally give you this url ![image](https://github.com/user-attachments/assets/05de7615-7486-47b5-9760-c740f15daa49)
Directly at the right bottom corner. 

https://xxx.integrationsuite.cfapps.eu10.hana.ondemand.com/ is right. 

Try to create a url in this style.
